recipe/meta.yaml

```diff
@@ -16,36 +18,28 @@ build:
     - pdfstream = pdfstream.main:main
     - run_server = pdfstream.main:run_server
     - print_server_config_dir = pdfstream.main:print_server_config_dir
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8, <3.10
     - pip
   run:
-    - python >=3.6
+    - python >=3.8, <3.10
     - bluesky
-    - databroker
+    - databroker =1.2
```

Comment:
I don't like this hard pinning. It won't work with the latest conda environments that use `databroker` 2+ and `tiled`. Could you please make sure the package is compatible with the new version of the library?

Reply:
There is a dependency issue in the `tiled`, addressed below.

https://github.com/bluesky/tiled/issues/258

I pin the version because of this issue. Garrett said that I could first pin the version and release it.